bin/deploy_automation/_global_vars.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+export PROD_BRANCH="prod"
+export PROD_ENV="prod"
+export STAGING_BRANCH="staging"
+export STAGING_ENV="staging"
+export MASTER_BRANCH="master"
+export GITHUB_TOKEN; GITHUB_TOKEN=${GITHUB_TOKEN:-$(cut -f 3 -d : ~/.git-credentials | cut -f 1 -d @)}
```

Comment:
I see `~/.git-credentials` here but export to `~/.bash_profile` listed in the Quip doc?

Reply:
The `${GITHUB_TOKEN` part is actually trying to read it from environment variable (set in `~/.bash_profile`). In case that variable is unset, then it defaults to that other part `:-$(cut ...)`